demos/python_demos/action_recognition/action_recognition.py

```diff
@@ -28,15 +28,18 @@
 
 sys.path.append(path.join(path.dirname(path.dirname(path.abspath(__file__))), 'common'))
 import monitors
+from images_capture import open_images_capture
 
 
 def build_argparser():
     parser = ArgumentParser(add_help=False)
     args = parser.add_argument_group('Options')
     args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
-    args.add_argument('-i', '--input',
-                      help='Required. Id of the video capturing device to open (to open default camera just pass 0), '
-                           'path to a video', required=True, type=str)
+    args.add_argument('-i', '--input', required=True,
+                      help='Required. An input to process. The input must be a single image, '
+                           'a folder of images or anything that cv2.VideoCapture can process')
+    args.add_argument('-loop', '--loop', default=False, action='store_true',
```

Comment:
Why is there a single-dash version of this option? A single dash should only be used for short options, which this isn't.

Reply:
Done.